app/src/org/commcare/android/logging/ForceCloseLogEntry.java

```diff
@@ -26,6 +26,7 @@ public class ForceCloseLogEntry extends AndroidLogEntry {
     public static final String STORAGE_KEY = "forcecloses";
 
     private int appBuildNumber;
+    // TODO: Drop android_version and device_model as these are part of the report header now
```

Comment:
remove this, all todos should strictly be in Jira where they can be tracked.

Reply:
[ab372b2](https://github.com/dimagi/commcare-android/pull/2977/commits/ab372b2df889091091fe6362d24a167dd4619e3a)